src/sparsezoo/model/utils.py

```diff
@@ -123,23 +112,53 @@ def load_files_from_stub(
     if isinstance(stub, str):
         stub, params = parse_zoo_stub(stub=stub, valid_params=valid_params)
     _LOGGER.debug(f"load_files_from_stub: loading files from {stub}")
-    response = download_get_request(
-        args=stub,
-        force_token_refresh=force_token_refresh,
+
+    arguments = get_model_metadata_from_stub(stub)
+
+    api = GraphQLAPI()
+
+    models = api.fetch(
+        operation_body="models",
+        arguments=arguments,
+        fields=["modelId", "modelOnnxSizeCompressedBytes"],
```

Comment:
I think it could be nice if these fields could be snake case. Then we have more opportunity to encapsulate all the camelCaseNess in the GraphQL code and the rest of the repo can have no idea that's going on

Reply:
fields can be camel or snake case!